docs/pull_request_template.md

```diff
@@ -18,4 +18,5 @@ _Any tips and tricks, blog posts or tools which helped you. Plus anything notabl
 * [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
 * [ ] I have added tests to prove my work
 * [ ] I have added mandatory tags to terraformed resources, where possible
+* [ ] If i added or updated a package.json or composer.json, I also made sure this is in the script in `.github/workflows/dependabot-update.yml`
```

Comment:
Is the wording of this right on this? From looking at the above I wouldn't have thought we need to add if it's updated, only if a new location is added.

Reply:
it's more of a sanity check to see if we've included it. i'll take updated out.

Reply:
```suggestion
* [ ] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
```